test/functional/core/job_spec.lua

```diff
@@ -115,6 +115,19 @@ describe('jobs', function()
     ok(string.find(err, "E475: Invalid argument: expected valid directory$") ~= nil)
   end)
 
+  it('produces error when using non-executable `cwd`', function()
+    if iswin() then return end  -- N/A for Windows
+
+    local dir = 'Xtest_not_executable_dir'
+    mkdir(dir)
+    helpers.call('system', {'chmod', '0600', dir})
```

Comment:
Why not use nvim's `setfperm(..., 'rw-------')`?

Reply:
Because I copied that from another test 🤭

I'll use `setfperm()` because it looks cleaner.